dev-util/cutter/cutter-1.1.recipe

```diff
@@ -0,0 +1,117 @@
+SUMMARY="A Qt and C++ GUI for radare2 reverse engineering framework"
+DESCRIPTION="Radare is a portable reversing framework that can...
+
+* Disassemble (and assemble for) many different architectures
+* Debug with local native and remote debuggers (gdb, rap, webui, r2pipe, \
+winedbg, windbg)
+* Run on Linux, *BSD, Windows, OSX, Android, iOS, Solaris and Haiku
+* Perform forensics on filesystems and data carving
+* Be scripted in Python, Javascript, Go and more
+* Support collaborative analysis using the embedded webserver
+* Visualize data structures of several file types
+* Patch programs to uncover new features or fix vulnerabilities
+* Use powerful analysis capabilities to speed up reversing
+* Aid in software exploitation"
+HOMEPAGE="https://github.com/radareorg/cutter/"
```

Comment:
```diff
-HOMEPAGE="https://github.com/radareorg/cutter/"
+HOMEPAGE="https://github.com/radareorg/cutter"
```
(The trailing slash is not needed here.)

I haven't tested the recipe, but everything else looks OK :)